src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -209,17 +220,14 @@ export default function Banner() {
     <React.Fragment>
       <CssBaseline />
       <div className="heroBanner">
-        <RetrieveBackgroundImage />
+        <RetrieveBannerDynamicAssets />
 
         <ThemeProvider>
           <Typography variant="h1" className={classes.h1}>
             {'Telescope'}
```

Comment:
We should file a bug to have this name come from the `gatsby-config.json` via our sitemetadata hook so it is configurable on another install.